quarkus-calculator/start-locally.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/sh
```

Comment:
Do we have a script like this for the quarkus-conference app? Or what other way would you recommend to start all the microservices? I'm tempted to use an analogous thing for the lab.

Reply:
We AFAIK, but we can (should) create it.